### DISCUSSION TITLE
EmbedIO with .NET Maui in conjunction with IOS

### DISCUSSION THREAD

**Discussion by manuelmair1986** [unosquare/embedio#592] [General]
Are there problems running EmbedIO in Maui in conjunction with IOS? Can the application that starts a local host with EmbedIO be loaded into the Apple Appstore? Because I did some tests and I wasn't able to start EmbedIO in conjunction with Maui on IOS.

**Comment by LRufenacht** [comment]
We accomplished this successfully for our platform, Wisej.NET Hybrid:
https://docs.wisej.com/Hybrid
https://github.com/unosquare/embedio/discussions/589

Apps have been successfully published to both the Android and iOS app stores (Search Wisej.NET on app store).
